templates/ts-apollo-postgres-backend/package.json

```diff
@@ -31,6 +31,7 @@
     "@types/glob": "7.1.2",
     "@types/graphql": "14.2.3",
     "@types/node-fetch": "2.5.7",
+    "graphback-cli": "0.14.0-alpha4",
```

Comment:
Actually, now that releases are automated through GitHub, templates will need to be manually adjusted (since I don't think renovate bumps alphas).

So this should be 0.14.0-alpha7?

Reply:
Thought so but I decided to align the version with other graphback packages version. 

Maybe @wtrocki can help here about how the process works?

Reply:
I imagine this problem will rectify itself once we are back on stable release, because templates probably should not be alpha/dev under production circumstances.

Reply:
Templates are par of the lerna and they are symlinked. Make sure you do lerna bootstrap and verify it to confim/

Reply:
Actually they are symlinked only when they are aligned so @machi1990 is perfect! fixing this issue